modules/n1ql/pages/n1ql-language-reference/statistics-delete.adoc

```diff
@@ -32,14 +32,12 @@ Deleting all statistics for a keyspace turns off the cost-based optimizer for al
 
 == Syntax
 
-[subs="normal"]
+[source,ebnf]
 ----
-update-statistics-delete ::=
-  ( UPDATE STATISTICS [ FOR ] | ANALYZE [ KEYSPACE | COLLECTION ] )
-    <<keyspace-ref>> <<delete-clause>>
+include::partial$grammar/utility.ebnf[tag=update-statistics-delete]
 ----
 
-image::n1ql-language-reference/update-statistics-delete.png["( 'UPDATE' 'STATISTICS' 'FOR'? | 'ANALYZE' ( 'KEYSPACE' | 'COLLECTION')? ) keyspace-ref delete-clause"]
```

Comment:
Previous contents were ALT text?
(I don't think we're very consistent with those, probably another one for a style-guide discussion sometime...)

Reply:
The content of the EBNF code block above _is_ effectively the alt text for these syntax diagrams. I should probably add alt text to the images saying so.